ros/processor/insights_engine_result_consumer.py

```diff
@@ -115,7 +128,42 @@ def process_report(self, host, reports):
                     fqdn=host['fqdn'],
                     rule_hit_details=reports,
                     number_of_recommendations=rec_count,
-                    state=SYSTEM_STATES[state_key]
+                    state=SYSTEM_STATES[state_key],
+                    instance_type=performance_record.get('instance_type')
+                )
+                LOG.info(
+                    f"{self.prefix} - System created/updated successfully: {host['id']}"
+                )
+
+                # For Optimized state, reports would be empty, but utilization_info would be present
+                if reports:
+                    set_default_utilization = True if reports[0].get('key') == 'NO_PCP_DATA' else False
+                else:
+                    set_default_utilization = False
```

Comment:
@apuntamb This was the only new change post our discussions, hope I'm making sense here

Reply:
Yeah +1